src/main/java/uk/gov/hmcts/reform/juddata/scheduler/JrdLeafDataLoadStarter.java

```diff
@@ -29,12 +29,22 @@
 
     private TaskScheduler scheduler;
 
+    @Value("${scheduler-name}")
+    private String schedulerName;
+
     @PostConstruct
     public void postConstruct() throws Exception {
         camelContext.start();
+        setGlobalOption();
         leafTableRoutes.startRoute();
     }
 
+    public void setGlobalOption() {
+        camelContext.getGlobalOptions().put(uk.gov.hmcts.reform.juddata.camel.util.MappingConstants.ORCHESTRATED_ROUTE, uk.gov.hmcts.reform.juddata.camel.util.MappingConstants.JUDICIAL_USER_PROFILE_ORCHESTRATION);
```

Comment:
get this camelContext.getGlobalOptions() in one variable and then use. Also use static constants for  MappingConstants